etc/krakend.json

```
@@ -85,7 +94,8 @@
           ],
           "extra_config": {
             "backend/http": {
-              "return_error_code": true
+              "return_error_code": true,
+              "return_error_details": "enrollment_service"
```

```suggestion
              "return_error_code": true
```
Because we want the Gateway to forward the Error codes from the Backends, we need to removed this line.

When you use return_error_details, all status codes returned to the client are 200.
Read more: https://www.krakend.io/docs/backends/detailed-errors/#return-backend-errors-in-a-new-key